src/components/DataTable/TableRow.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import omit from 'lodash.omit';
+import classNames from 'classnames';
```

Comment:
Could we use the `cx` convention for the name here? e.g. `import cx from 'classnames'`